pinot-common/src/main/java/org/apache/pinot/pql/parsers/pql2/ast/ComparisonPredicateAstNode.java

```diff
@@ -155,6 +157,74 @@ public FilterQueryTree buildFilterQueryTree() {
     }
   }
 
+  @Override
+  public Expression buildFilterExpression() {
```

Comment:
Are we going to support function in comparison?

Reply:
yes. I have a PR  that supports UDFs https://github.com/winedepot/pinot/pull/28 in filter clause